ui/src/state/settings.ts

```diff
@@ -291,3 +320,16 @@ export function useBrowserNotifications(browserId: string): boolean {
   const browserSetting = getBrowserSetting(settings, browserId);
   return browserSetting?.browserNotifications ?? false;
 }
+
+export function useGroupsHasBeenUsed() {
+  const { data, isLoading } = useGroupsSettings();
+
+  return useMemo(() => {
+    if (isLoading || data === undefined) {
+      return false;
```

Comment:
let's default to not showing until we know for sure

```suggestion
      return true;
```